docs/pages/project/compatibility_matrix/compatibility.md

```diff
@@ -2,6 +2,7 @@
 layout: default
 title: Compatibility Matrix
 permalink: project/compatibility-matrix
+redirect_from: /
```

Comment:
This doesn't seem to fix the issue, can you try changing `/` to `project/compatibility-matrix/`

Reply:
I have Updated it can you please check it @asubedy

Reply:
Yes this works 👍